packages/react-popover/src/components/PopoverTrigger/renderPopoverTrigger.tsx

```diff
@@ -1,8 +1,9 @@
+import * as React from 'react';
 import type { PopoverTriggerState } from './PopoverTrigger.types';
 
 /**
  * Render the final JSX of PopoverTrigger
  */
-export const renderPopoverTrigger_unstable = (state: PopoverTriggerState): JSX.Element => {
-  return state.children;
+export const renderPopoverTrigger_unstable = (state: PopoverTriggerState) => {
+  return <>{state.children}</>;
```

Comment:
Not sure that it worth a comment, but as `state.children` can be `React.ReactNode` (that includes arrays and `undefined`) we should wrap this with fragment.

P.S. It can be `React.ReactNode` because we allow this in render props.

Reply:
https://github.com/microsoft/fluentui/blob/165a46a8e5fbbe4ac8356666a8df6ab95a09ceb7/packages/react-tooltip/src/components/Tooltip/Tooltip.types.ts#L107-L110

https://github.com/microsoft/fluentui/blob/165a46a8e5fbbe4ac8356666a8df6ab95a09ceb7/packages/react-menu/src/components/MenuTrigger/MenuTrigger.types.ts#L7

¯\_(ツ)_/¯

Reply:
ReactNode can also be `string` and `boolean` which seems like things we definitely don't want as a child

Reply:
Honestly, I vote for `React.ReactElement | null`. For me it makes more sense in this scenario.

Reply:
why `null` ? is there a reason we just want to render null inside a trigger ?

Reply:
I used the `ReactNode` type when I added support for render functions in Tooltip. Tooltip had a restrictive requirement that it must have a single child that accepts HTMLElement props. So as an escape hatch I added the render function that could return anything that would have been valid for the `children` prop of any other component (ReactNode).

That said, if the render function only allowed `ReactElement | null`, the render function could just return a Fragment around whatever it needed. So it seems like a fine change to make for all of our trigger elements.

Reply:
So, what is conclusion? 🙂 Should I update every render props function to have `ReactElement | null` as return value?

Reply:
👍 sounds good to me. 

Maybe even make a `TriggerRenderFunction` type to ensure consistency? (It's fine not to add the type if you don't think it's worthwhile).

Reply:
Agreed, I will update types on Monday for all triggers.

Reply:
Changes applied in aa7687b.

Reply:
I also restored the fragment change, 3096924.